solidity/dashboard/src/sagas/subscriptions.js

```diff
@@ -759,6 +767,27 @@ function* observeWrappedTokenMintAndBurnTx(liquidityRewardPair) {
       if (from === ZERO_ADDRESS || to === ZERO_ADDRESS) {
         yield* updateAPY(LPRewardsContract, liquidityRewardPair.name)
       }
+
+      // If the 'to' address is equal to the address of the connected wallet
+      // then it means that LP tokens are transferred to this address so we are
+      // displaying the proper notification
+      if (to === defaultAccount) {
+        yield put(
```

Comment:
I think should we also update the [`wrappedTokenBalance`](https://github.com/keep-network/keep-core/blob/master/solidity/dashboard/src/reducers/liquidity-rewards.js#L8) for a given liquidity rewards pair.